doc/Confluent_Platform_Setup.md

```diff
@@ -0,0 +1,96 @@
+# Confluent Platform Setup
+
+This guide walks through setting up Confluent Platform using Docker containers.
+
+## Prerequisites
+
+- Bash shell (tested on Github Codespaces, Mac, Ubuntu, Windows with WSL2)
+  - Will not work in Cloud Shell or WSL1
+- Java 11+ ([download](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html))
+- Maven ([download](https://maven.apache.org/download.cgi))
+- Docker ([download](https://www.docker.com/products/docker-desktop))
+- Powershell (optional) ([download](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell))
+
+## Setup
+
+> [Github Codespaces](https://github.com/features/codespaces) is the easiest way to evaluate the Cosmos DB Kafka Connectors as all of the prerequisites are automatically installed
+>
+> Follow the setup steps in the [developer setup](../DEVELOPER_WALKTHROUGH.MD) to setup Codespaces
+
+### Initialize repo
+
+Clone the Kafka Connect Cosmos DB repo
+
+```bash
+
+### skip this step if using Codespaces
+
+git clone https://github.com/microsoft/kafka-connect-cosmosdb.git
+
+cd kafka-connect-cosmosdb
+export REPO_ROOT=$(pwd)
+
+```
+
+### Startup
+
+Start up the docker containers for Confluent Platform using `docker-compose`
+
+```bash
+
+cd $REPO_ROOT/src/integration-test
+
+# If you're using codespaces, either option will work.
```

Comment:
i would put these sort of comments / instructions in the actual md text, not in the code sample.